src/svc/invocable-scripts/schemas/ScriptAddLabelsRequest.yml

```diff
@@ -0,0 +1,7 @@
+type: object
+properties:
```

Comment:
Can we have a list of label names object that used for both adding and removing?